src/dom/DomEvent.js

```diff
@@ -23,8 +23,17 @@ var DomEvent = /** @lends DomEvent */{
             for (var type in events) {
                 var func = events[type],
                     parts = type.split(/[\s,]+/g);
-                for (var i = 0, l = parts.length; i < l; i++)
-                    el.addEventListener(parts[i], func, false);
+                for (var i = 0, l = parts.length; i < l; i++) {
+                    var eventName = parts[i];
+                    // For touchstart/touchmove events on document, we need to explicitely
+                    // declare that event is not passive (can be prevented).
+                    // Otherwise chrome browser would ignore event.preventDefault() calls.
+                    // See #1501 and https://www.chromestatus.com/features/5093566007214080
+                    var options = el === document && (eventName === 'touchstart'|| eventName === 'touchmove')
+                                  ? {passive: false}
+                                  : false;
```

Comment:
Some code-style remarks, to align with the paper.js code-style:

- One tab-indent before the `?`, not aligned with `=`
- Spaces inside objects: `{ passive: false }`
- The line looks longer than 80 chars, so it should be broken. Maybe `name` instead of `eventName` would help a little?

I should maybe install ESLint and / or Prettier, so we have a more modern linter and less discussions about code-style :)